Examples/UtiliKit-iOSExample/ViewController/InitialViewController.swift

```diff
@@ -6,15 +6,20 @@
 //
 
 import UIKit
+import UtiliKit
 
 /**
  A simple view controller used in the example showcasing how to programmatically instantiate view controllers.
  
  Check out VCViewController and UIStoryboardIdentifier+Extensions.swift to see how the view controllers and storyboards are instantiated in a type-safe fashion.
  */
-class InitialViewController: UIViewController {
+class InitialViewController: UIViewController, Configurable {
     
     @IBAction func didSelectClose(_ sender: Any) {
         dismiss(animated: true)
     }
+    
+    func configure(with element: String) {
+        /* No op */
```

Comment:
We should probably do something here with the string we passed in like showing it in a label. Or let's right a better comment.